packages/react-components/src/index.ts

```diff
@@ -32,6 +32,9 @@ export { useTimeSeriesData } from './hooks/useTimeSeriesData';
 export { useHasFeatureFlag } from './hooks/useHasFeatureFlag';
 export { useGetConfigValue } from './store/index';
 
+export { useAlarms } from './hooks/useAlarms';
```

Comment:
Is there any way we can avoid exporting useAlarms? Can we just use the alarm query directly and display the assetId / compositeModelId for now?

Reply:
yes! will update as a followup